tests/components/asuswrt/test_sensor.py

```diff
@@ -39,13 +39,16 @@
 
 MOCK_BYTES_TOTAL = [60000000000, 50000000000]
 MOCK_CURRENT_TRANSFER_RATES = [20000000, 10000000]
+MOCK_LOAD_AVG = [1.1, 1.2, 1.3]
 
 SENSOR_NAMES = [
     "Devices Connected",
     "Download Speed",
     "Download",
     "Upload Speed",
     "Upload",
+    "Load Avg (1m)",
+    "Load Avg (15m)",
```

Comment:
Missing 5 min?

Reply:
I do not enabled the 5 min sensor and do not test it, but I can add, not a problem. Just think that if first and last was ok, also the middle should be ok😁

Reply:
🤷 Just a couple of lines to add that coverage, I would just do that :)